crates/red_knot_python_semantic/resources/mdtest/intersection_types.md

```diff
@@ -422,8 +422,8 @@ def example_type_bool_type_str(
 
 #### Positive and negative contributions
 
-If we intersect a type `X` with the negation of a disjoint type `Y`, we can remove the negative
-contribution `~Y`, as it necessarily overlaps with the positive contribution `X`:
+If we intersect a type `X` with the negation `~Y` of a disjoint type `Y`, we can remove the negative
+contribution `~Y`, as `~Y` must fully contain the positive contribution `X` as a subtype:
```

Comment:
@carljm You phrased it as "subset". Is it also correct so say "subtype" here?

As in, could we write a property test `∀ X, Y. is_disjoint_from(X, Y) => is_subtype_of(X, ~Y)`?

Reply:
Yes, subtype is accurate as well! And yes, that property test should be valid.